### DISCUSSION TITLE
[GitHub Actions] Unknown (empty) errors in build logs

### DISCUSSION THREAD

**Discussion by aiartefact** [community/community#33698] [Actions] [answered]
I'm running a simple gtest project built with cmake with the following yml-file:

![yml-file](https://user-images.githubusercontent.com/71671107/191510668-432ba22f-ebbd-4454-adb0-4270f9875138.png)

Build is successful but logs are not visible due to some unknown (empty) error:

![log-error](https://user-images.githubusercontent.com/71671107/191508536-09edc255-1d03-4c54-96fb-8a863461455c.png)

However, raw logs indicate successful build completion and contain successful output from gtest:

![raw-log-outtake](https://user-images.githubusercontent.com/71671107/191509352-5001a05c-78d8-4858-b983-2a1483f9abb7.png)

It seems that accessing raw logs every time should "solve" the problem, but at least it is inconvenient. Besides, those empty error messages indicate some kind of a problem. 
So, my questions are: What's wrong with conventional logs and how to fix it?

**Comment by airtower-luna** [comment, marked as answer]
I've seen similar with strict mode in uBlock Origin, where I was blocking requests to other domains by default. I had to allow requests to the server the logs are loaded from (`pipelines.actions.githubusercontent.com`, which is a CNAME for `l-0007.l-msedge.net`).


If you have any kind of adblocker or similar, check if it's blocking requests when you try to load the logs.

**Reply by aiartefact** [reply]
Thank you very much for the idea where to search! I haven't searched in that direction because my AdGuard was disabled for GitHub. But your answer gave me idea to double-check every single extension. The problem was hiding in NoScript extension. Its icon indicated all scripts to be enabled, but ` ...actions.githubusercontent.com` was marked as blocked. As soon as it was enabled logs became visible:

![logs-work-properly](https://user-images.githubusercontent.com/71671107/191740545-00218d18-c07d-4fed-beaf-63e7c49f25d1.png)

For anyone who will ever face similar problem I suggest to double-check every single extension in the browser and network activity.
